debian/postrm

```diff
@@ -18,24 +18,19 @@ set -e
 # the debian-policy package
 
 case "$1" in
-
     remove|purge)
-        grep -v 'zabbix-check-sshkey' /etc/sudoers > /etc/.sudoers
-        mv /etc/.sudoers /etc/sudoers
-        chmod 440 /etc/sudoers
```

Comment:
If you now take it out, it will not be removed when updating and will therefore appear twice.

Reply:
Oops, you are right. I thought this was some historical stuff and removed it without checking...

Reply:
Done